indra/newview/llfloateremojipicker.cpp

```diff
@@ -91,19 +100,41 @@ LLFloaterEmojiPicker::LLFloaterEmojiPicker(const LLSD& key)
 
 BOOL LLFloaterEmojiPicker::postBuild()
 {
-	if ((mEmojis = getChild<LLScrollListCtrl>("Emojis")))
+	// Should be initialized first
+	if ((mPreviewEmoji = getChild<LLButton>("PreviewEmoji")))
```

Comment:
getChild normally does not return null, it will either find the element, create a dummy or we should crash due to a missing widget.

Reply:
> getChild normally does not return null, it will either find the element, create a dummy or we should crash due to a missing widget (aka LLButton, if we don't have that, we have problems).

We've discussed this, and I specially took a look at the source code
According to the source code, _theoretically_ NULL is possible
And who knows what would be changed in getChild in future?
There is no problem for me to check this field, just for assurance

Reply:
If that theoretical scenario happens, it deserves to crash.